src/grokcore/content/tests/test_container.py

```diff
@@ -18,15 +18,15 @@
 this doesn't happen again:
 
     >>> skull = Bone()
-    >>> print skull.__parent__
+    >>> print(skull.__parent__)
     None
-    >>> print skull.__name__
+    >>> print(skull.__name__)
     None
     >>> bag['skull'] = skull
     >>> skull.__parent__
     <grokcore.content.tests.test_container.BoneBag object at 0x...>
-    >>> skull.__name__
-    u'skull'
+    >>> 'skull name';skull.__name__
```

Comment:
What about `str(skull.__name__)` this has the same `repr` in Python 2 and 3 of `'skull'`.